.projenrc.ts

```diff
@@ -3,15 +3,15 @@ import { NpmAccess } from 'projen/lib/javascript';
 import { WorkflowNoDockerPatch } from './projenrc/workflow-no-docker-patch';
 
 // the version of k8s this branch supports
-const SPEC_VERSION = '26';
+const SPEC_VERSION = '27';
 const releaseWorkflowName = `release-kubectl-v${SPEC_VERSION}`;
 const defaultReleaseBranchName = `kubectl-v${SPEC_VERSION}/main`;
 
 const project = new awscdk.AwsCdkConstructLibrary({
   projenrcTs: true,
   author: 'Amazon Web Services',
   authorAddress: 'aws-cdk-dev@amazon.com',
-  cdkVersion: '2.0.0',
+  cdkVersion: '2.83.0',
```

Comment:
this is minimum cdk version compatible with this project, i dont think we want to change it.